pkg/acsengine/const.go

```diff
@@ -57,9 +57,9 @@ var KubeImages = map[api.OrchestratorVersion]map[string]string{
 		"exechealthz":  "exechealthz-amd64:1.2",
 		"addonresizer": "addon-resizer:1.6",
 		"heapster":     "heapster:v1.2.0",
-		"dns":          "kubedns-amd64:1.7",
+		"dns":          "k8s-dns-kube-dns-amd64:1.13.0",
```

Comment:
Yes, not sure why, but that's the name:

https://console.cloud.google.com/gcr/images/google-containers/GLOBAL/k8s-dns-kube-dns-amd64